deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -300,12 +284,81 @@ spec:
         name: statefulsets.apps
         version: v1
       version: v1alpha1
-  description: |2-
+  description: |2
+
+    Red Hat Openshift Container Storage (RHOCS) provides hyperconverged storage for applications within an Openshift cluster.
+
+    ## Components
+
+    RHOCS deploys 3 operators.
+
+    ### OCS operator
+
+    The OCS operator is the primary operator for RHOCS. It serves to facilitate the other operators in OCS by performing administrative tasks outside their scope as well as watching and configuring their CustomResources.
+
+    ### Rook
+
+    Rook[1][1] deploys and manages Ceph on Openshift, which provides block and file storage.
+
+    ### NooBaa operator
+
+    The NooBaa operator deploys and manages NooBaa[2][2] on Openshift, which provides object storage.
+
+    ## Before Subscription
+
+    Before subcribing to RHOCS, there are two pre-requisites that need to be satisfied.
+
+    ### Namespace
+
+    RHOCS runs only in the openshift-storage Namespace, which needs to be created before subscription. The following manifest can be used to create the Namespace.
+
+    ```
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      labels:
+        openshift.io/cluster-monitoring: "true"
+      name: openshift-storage
+    spec: {}
+    ```
+
+    Save the above as rhocs-namespace.yaml, and create the Namespace with,
+
+    ```
+    $ oc create -f rhocs-namespace.yaml
+    ```
+
+
+    ### OperatorGroup
```

Comment:
Shouldn't the OLM handle the creation of the OperatorGroup?

Reply:
Nope. The OperatorGroup is created by the user. OLM installs into the operator group.

Reply:
Is it the same with all the operators?